### PR TITLE
Locales: Update Translations (fr_FR)

### PR DESCRIPTION
**Pull request by LaneSh4d0w**
As discussed over Discord:

- Fixed furthermore the poorly translated lines
- Deleted the obvious translations

### CODE REVIEW

File: locales/fr_FR/pcsx2_Main.po [hunk 1]
```diff
@@ -113,21 +115,22 @@ msgid ""
 "image type or a bug in PCSX2 ISO image support."
 msgstr ""
 "Si cette erreur se produit à la suite du montage d'une image ISO, c'est que "
-"le format utilisé semble ne pas être pris en charge par PCSX2."
+"le format semble ne pas être pris en charge par PCSX2, ou bien d'un bug lié "
```
Comment (Rackover) [inline locales/fr_FR/pcsx2_Main.po:118]:
```suggestion
"le format semble ne pas être pris en charge par PCSX2, ou bien il s'agit d'un bug lié "
```

File: locales/fr_FR/pcsx2_Main.po [hunk 2]
```diff
@@ -174,29 +177,28 @@ msgstr ""
...
 "supportés."
 
 #: pcsx2/PluginManager.cpp:1031
 msgid ""
 "Configured plugin is not a PCSX2 plugin, or is for an older unsupported "
 "version of PCSX2."
 msgstr ""
-"Le plugin configuré n'est pas un plugin PCSX2, ou la version utilisée est "
-"trop ancienne."
+"Le plugin configuré n'est pas un plugin PCSX2, ou pour une version trop "
```
Comment (Rackover) [inline locales/fr_FR/pcsx2_Main.po:188]:
```suggestion
"Le plugin configuré n'est pas un plugin PCSX2, ou conçu pour une version trop "
```

File: locales/fr_FR/pcsx2_Main.po [hunk 3]
```diff
@@ -345,20 +347,20 @@ msgstr "Bloc-note de l'activité MFIFO."
 #: pcsx2/SourceLog.cpp:258
 msgid "Actual data transfer logs, bus right arbitration, stalls, etc."
 msgstr ""
-"Données réelles des transferts de rapports, vrai arbitrage du bus, stalls, "
-"etc."
+"Données réelles des transferts de rapports, arbitrage réel du bus, "
+"stabulation, etc."
```
Comment (Rackover) [inline locales/fr_FR/pcsx2_Main.po:351]:
```suggestion
"retards, etc."
```
Je ne suis pas sûre que "stabulation" soit valide pour traduire "stall" dans un contexte informatique (il me semble que c'est un mot qu'on donne plutôt au bétail). stall pourrait plutôt être traduit par "blocage", "retard", "décrochage" je pense.

File: locales/fr_FR/pcsx2_Main.po [hunk 4]
```diff
@@ -377,7 +379,7 @@ msgstr "/DISASM des instructions du co-processeur GPU IOP."
 msgid ""
 "All known hardware register accesses, not including the sub-filters below."
 msgstr ""
-"Accès à tous les registres hardware connus, à l'exception des filtres ci-"
+"Accès des registres hardware connus, à l'exception des sous-filtres ci-"
```
Comment (Rackover) [inline locales/fr_FR/pcsx2_Main.po:382]:
```suggestion
"Accès à tous les registres hardware connus, à l'exception des sous-filtres ci-"
```

File: locales/fr_FR/pcsx2_Main.po (hunk 3 above)
Comment (LaneSh4d0w) [reply locales/fr_FR/pcsx2_Main.po:351]: J'hésite encore avec décrochage, car effectivement, quand on a pas le contexte initial...

Comment (LaneSh4d0w) [reply locales/fr_FR/pcsx2_Main.po]: Modifié pour "blocage"

**Comment by LaneSh4d0w** [comment]
> 
> 
> Je suis pas dev ici, mais je suis française et je passais par là alors j'ai fait quelques propositions :)

Merci à toi d'en proposer :)

**Comment by LaneSh4d0w** [comment]
Translations purged. Review permitted to make some fixes into the translations.

File: locales/fr_FR/pcsx2_Iconized.po [hunk 5]
```diff
@@ -877,225 +883,7 @@ msgid ""
 "not a critical error, since the sVU rec is obsolete, and you should use "
 "microVU instead anyway. :)"
 msgstr ""
-"Out of Memory (ou presque) : le recompilateur SuperVU n'est pas parvu à "
+"Out of Memory (ou presque) : le recompilateur SuperVU n'est pas parvenu à "
 "allouer suffisamment de mémoire virtuelle, ce qui rend impossible son "
-"utilisation. Ce n'est pas une erreur critique, puisque le recompilateur sVU "
-"est obsolète et puisque vous devriez utiliser microVU ! :p"
```
Comment (RedDevilus) [inline locales/fr_FR/pcsx2_Iconized.po:883]: Removed a bit too much? Line 882-884. 
Also Warning that endline breaks.
image

File: locales/fr_FR/pcsx2_Main.po [hunk 6]
```diff
@@ -3009,227 +3014,4 @@ msgid ""
...
-
-#~ msgid "mVU Block Hack"
-#~ msgstr "mVU Block Hack"
-
-#~ msgid ""
-#~ "Good Speedup and High Compatibility; may cause bad graphics, SPS, etc..."
-#~ msgstr ""
-#~ "Amélioration sensible des performances et compatibilité élevée, mais "
-#~ "pourrait causer quelques bugs graphiques, SPS, etc..."
+"extensions SSE2."
```
Comment (RedDevilus) [inline locales/fr_FR/pcsx2_Main.po:3017]: Same here Newline breaking. Guess ill have to test to make sure it doesnt break.

File: locales/fr_FR/pcsx2_Iconized.po (hunk 5 above)
Comment (LaneSh4d0w) [reply locales/fr_FR/pcsx2_Iconized.po:883]: Probably just GitHub being a bit lazy here, removed nothing more than the old translations.

File: locales/fr_FR/pcsx2_Main.po (hunk 6 above)
Comment (LaneSh4d0w) [reply locales/fr_FR/pcsx2_Main.po:3017]: It normally shouldn't, I suppose that it might put it back if you open the PO file...

Comment (RedDevilus) [reply locales/fr_FR/pcsx2_Main.po:3017]: I'll wait till appveyor is done and test myself and if it works i'll blame GitHub. 😉

File: locales/fr_FR/pcsx2_Iconized.po (hunk 5 above)
Comment (turtleli) [reply locales/fr_FR/pcsx2_Iconized.po:883]: It means that the last characters in the file is not a newline sequence.

**Comment by RedDevilus** [comment]
Only missing a hotkey for Quitter. But thats nitpicky.

**Comment by lightningterror** [comment]
Is it ready for merge?

**Comment by RedDevilus** [comment]
It looks great otherwise. I would say you could merge this as nothing is broken. (minus what i said about the hotkey which still works)

File: locales/fr_FR/pcsx2_Main.po [hunk 7]
```diff
@@ -3009,227 +3014,4 @@ msgid ""
 "%s Extensions not found.  microVU requires a host CPU with SSE2 extensions."
 msgstr ""
 "%s extensions non trouvées.  microVU nécessite un CPU qui gère les "
-"extensions SSE2."
```
Comment (lightningterror) [inline locales/fr_FR/pcsx2_Main.po:3012]: This line shouldn't be removed.

Comment (RedDevilus) [reply locales/fr_FR/pcsx2_Main.po:3012]: :) same as what i said above.

**Pull request merged**